### PR TITLE
SpruceEats was duplicate of TheSpruceEats, merged both, updated scraper, tests and README accordingly.

### PR DESCRIPTION
**Pull request by khushhalm**
There are two scrapers for the same [TheSpruceEats website](https://www.thespruceeats.com/).

- Merged and updated the scraper: TheSpruceEats.
- Removed files for 'SpruceEats'
- Updated unit tests and HTML files for test cases
- Updated README accordingly

To run the unit test for this particular scraper, you can use this: `python -m coverage run -m unittest tests.test_thespruceeats `

### CODE REVIEW

**Comment by khushhalm** [comment]
Files Added/Modified/Removed in these commits:

[modified] Python scraper, TheSpruceEats, code for the [website](https://www.thespruceeats.com/doner-kebab-recipe-4171703).
[removed] Python scraper, SpruceEats, code for the [website](https://www.thespruceeats.com/doner-kebab-recipe-4171703).
[modified] HTML test file for TheSpruceEats.
[removed] HTML test file for SpruceEats.
[modified] test_thespruceeats.py: Python unit test for TheSpruceEats.
[removed] test_spruceeats.py: Python unit test for SpruceEats.
[modify] The main recipe-scrapers Python module
[modify] README

**Comment by khushhalm** [comment]
@jayaddison @hhursev 
Can you please check this and merge this, if this looks good to you? If not, can you let me know if I missing running any test or something?

File: recipe_scrapers/__init__.py [hunk 1]
```diff
@@ -134,6 +133,7 @@
 from .thenutritiouskitchen import TheNutritiousKitchen
 from .thepioneerwoman import ThePioneerWoman
 from .thespruceeats import TheSpruceEats
```
Comment (hhursev) [inline recipe_scrapers/__init__.py:135]: Remove me

File: recipe_scrapers/__init__.py [hunk 2]
```diff
@@ -289,6 +288,7 @@
     TheNutritiousKitchen.host(): TheNutritiousKitchen,
     ThePioneerWoman.host(): ThePioneerWoman,
     TheSpruceEats.host(): TheSpruceEats,
```
Comment (hhursev) [inline recipe_scrapers/__init__.py:290]: Remove me

**Comment by hhursev** [comment]
Two duplicated lines for removal everything else is perfect 👍  🍻 Thanks for updating the test file!

I'll merge and bump version when you take care of the couple comments.

This closes #363

File: recipe_scrapers/__init__.py (hunk 2 above)
Comment (khushhalm) [reply recipe_scrapers/__init__.py]: Done [58146e6](https://github.com/hhursev/recipe-scrapers/pull/382/commits/58146e6af967de4f8b369c901f3d921e4da98d1c)

File: recipe_scrapers/__init__.py (hunk 1 above)
Comment (khushhalm) [reply recipe_scrapers/__init__.py]: Done [58146e6](https://github.com/hhursev/recipe-scrapers/pull/382/commits/58146e6af967de4f8b369c901f3d921e4da98d1c)

**Comment by khushhalm** [comment]
Hey, @hhursev thanks for reviewing it. I've resolved with the commit [58146e6](https://github.com/hhursev/recipe-scrapers/pull/382/commits/58146e6af967de4f8b369c901f3d921e4da98d1c)

**Comment by khushhalm** [comment]
The above commit is: Reformatted code by black so that it passes linters test.

**Comment by khushhalm** [comment]
Is there anything else that needs to be done to close this? @hhursev

**Comment by hhursev** [comment]
It's perfect :) Merging and bumping version

**Comment by jayaddison** [comment]
Nice, thanks @khushhalm!

**Comment by khushhalm** [comment]
Happy to be of some help @jayaddison :)

**Pull request merged**